www/css/cytube.css

```diff
@@ -239,8 +241,9 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 .server-msg-disconnect {
     border: 1px solid #ff0000;
     padding-top: 5px;
-    padding-bottom: 5px;
-    margin: 5px;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    height: 30px;
```

Comment:
Essentially what I'm confused about is why you're changing it with the goal of getting the same end result (the current height of the box).  Changing the `margin: 5px` to `margin-top` and `margin-bottom` makes sense, since with the chat messagebuffer padding it's no longer necessary to have left/right margins set on these messages.  However, making those changes only, I don't observe breakage of the height of the Connected box.